test_fixtures/gold_output_files/backwards_compatible/basic.over_react.g.dart.goldFile

```diff
@@ -173,7 +173,7 @@ class _$BasicComponent extends BasicComponent {
   bool get $isClassGenerated => true;
 
   /// The default consumed props, taken from _$BasicProps.
-  /// Used in `ConsumedProps` if [consumedProps] is not overridden.
+  /// Used in -`ConsumedProps` methods if [consumedProps] is not overridden.
```

Comment:
```suggestion
  /// Used in `ConsumedProps` methods if [consumedProps] is not overridden.
```

Reply:
Or feel free to ignore - if the `-` has meaning... just looks like a typo to me.

Reply:
Yeah, it was purposeful; I meant all methods ending in that string. How about `*ConsumedProps` instead?

Reply:
Addressed in e3f27c9c597555e0e3cb917a34687c9b9859fd57